### DISCUSSION TITLE
Welcome to CPM-Live Discussions!

### DISCUSSION THREAD

**Discussion by jayzzhou-thu** [OpenBMB/CPM-Live#1] [Announcements 通知]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. Here you can:

  * Get in touch with the community by reading **Announcements**
  *  Share your ideas on creative applications of CPM-Live in **Application Ideas**
  * Raise your **Model Proposals** to make CPM-Live better
  * Talk about everything in **Open Chats**
  * Ask questions you’re wondering about in **Q & A**
  * Read the **Training Logs**
  * Engage with other community members
  * Welcome others and are open-minded. Remember that this is a community we build together 💪
  * English is recommended (Chinese is also accepted)

  To get started, comment below with an introduction of yourself!

----
## 👋 欢迎！

 我们使用Discussions功能连接社区成员，在这里你可以：
  * 通过阅读 **Announcements 通知** 了解社区动态
  * 在 **Application Ideas 应用想法** 中分享你关于CPM-Live的创意应用想法
  * 提交你的 **Model Proposals 模型提议** 让CPM-Live变得更好
  * 在 **Open Chats 开放交流** 中谈论任何事情
  * 在 **Q & A 问答** 中提出你的问题
  * 阅读 **Training Logs 训练日志**
  * 和其他社区成员建立联系
  * 开放心态，迎接朋友，共同构建我们的社区 💪
  * 我们推荐使用英语（中文也会被响应）
  
 在下方评论，做一个自我介绍吧！

**Comment by frankwangtoronto** [comment]
firt spot ,  all the best to this project

**Comment by weiiWill** [comment]
so excited that all people interesed in Big model can make contributions and learn what they want by this kind of cooperation and sharing.

**Comment by Kunlun-Zhu** [comment]
Can't wait to see the next steps

**Comment by VivaLaDijkstra** [comment]
code is cheap, show the model parameters

**Comment by cherry00725** [comment]
不管用什么都显示#Error：服务 是什么情况
